File: python/bucles/registrar_contactos_con_while.py
contactos = []
i = 0
cantidad = int(input("¿Cuántos contactos desea registrar?: "))
while i < cantidad:
    print(f"\nContacto {i+1}:")
    nombre = input("Ingrese el nombre: ")
    telefono = input("Ingrese el teléfono: ")
    contactos.append((nombre, telefono))
    i += 1
print("Contactos registrados:")
for nombre, telefono in contactos:
    print("Nombre:", nombre, "- Teléfono:", telefono)

File: visual_basic/bucles/registrar_contactos_con_while/registrar_contactos_con_while.vb
Imports System
Module registrar_contactos_con_while
    Sub Main(args As String())
        Dim cantidad As Integer
        Dim i As Integer = 0
        Console.Write("¿Cuántos contactos desea registrar?: ")
        cantidad = Convert.ToInt32(Console.ReadLine())
        Dim contactos(cantidad - 1) As String
        While i < cantidad
            Console.Write("Ingrese el nombre del contacto #" & (i + 1) & ": ")
            contactos(i) = Console.ReadLine()
            i += 1
        End While
        Console.WriteLine(vbCrLf & "=== CONTACTOS REGISTRADOS ===")
        i = 0
        While i < cantidad
            Console.WriteLine((i + 1) & ". " & contactos(i))
            i += 1
        End While
    End Sub
End Module